### ISSUE TITLE
avoid globalThis shim when not needed?

### ISSUE THREAD

**Issue by dckc** [Agoric/agoric-sdk#63]
xs gripes:
```
...SwingSet/src/controller.js (24) # Exception: cannot coerce undefined to object!.
```

When I comment out the `unshift` line, xs is happy. I'm pretty sure it's not needed in xs; xs provides `globalThis` in every compartment. Can we skip this `unshift` on platforms where it's not needed?

https://github.com/Agoric/SwingSet/blob/e103ff770ca083987ff7b824057780ea1813d15d/src/controller.js#L22-L24

**Comment by dckc** [comment]
According to the design in #2107 , we're not running the controller on xs , for now.

**Comment by dckc** [comment]
fixed in 341718be335e16b5
